From d128c123754e9dd03ad72c16851a1652331d6da1 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 27 Nov 2024 10:24:47 +0000
Subject: [PATCH] dts: bcm2712-rpi-cm5: Remove inaccessible USB_OC_N

Although VBUS_EN on GPIO42 appears on the CM5's 100-way headers,
USB_OC_N on GPIO43 does not. Remove the signal name to avoid further
confusion and disappointment.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
@@ -718,7 +718,7 @@ spi10_cs_pins: &spi10_cs_gpio1 {};
 		"-", // GPIO40
 		"-", // GPIO41
 		"USB_VBUS_EN", // GPIO42
-		"USB_OC_N", // GPIO43
+		"-", // GPIO43
 		"RP1_STAT_LED", // GPIO44
 		"FAN_PWM", // GPIO45
 		"-", // GPIO46
